packages/itwinui-css/src/bridge.scss

```diff
@@ -250,16 +288,16 @@
     // StrataKit uses different surface colors for the dropdowns
     background-color: var(--stratakit-color-bg-elevation-level-1);
 
+    // Hover state background color
+    :is(.iui-list-item, .iui-menu-item-skeleton) {
```

Comment:
Is this nesting necessary or can it be applied at the top-level? (It will impact multiple components)